docs/database/mongodb-component.md

```diff
@@ -110,29 +114,18 @@ And then the connection string will be retrieved from the `ConnectionStrings` co
 }
 ```
 
-For more information on how to format this connection string, see [MongoDB: ConnectionString documentation](https://www.mongodb.com/docs/v3.0/reference/connection-string).
-
-### Use configuration providers
-
-The .NET Aspire MongoDB database component supports <xref:Microsoft.Extensions.Configuration?displayProperty=fullName>. It loads the `MySqlConnectorSettings` from configuration files such as _:::no-loc text="appsettings.json":::_ by using the `Aspire:MongoDB:Driver` key. If you have set up your configurations in the `Aspire:MongoDB:Driver` section, you can just call the method without passing any parameter.
-
-The following example shows an _:::no-loc text="appsettings.json":::_ file that configures some of the available options:
+MongoDB Atlas Eexmple:
 
 ```json
 {
-  "Aspire": {
-    "MongoDB": {
-      "Driver": {
-        "ConnectionString": "mongodb://server:port/test",
-        "DisableHealthChecks": false,
-        "HealthCheckTimeout": 10000,
-        "DisableTracing": false
-      },
-    }
+  "ConnectionStrings": {
+    "MongoConnection": "mongodb+srv://username:password@server.mongodb.net/",
   }
```

Comment:
Please revert these changes, I understand the need to show differences between connection strings for MongoDB and MongoDB with Atlas, but this configuration is still valid and needs to be reflected in docs.